pom.xml

```diff
@@ -84,6 +84,10 @@
         <!-- Models Repository -->
         <module>sdk/modelsrepository/azure-iot-modelsrepository</module>
 
+        <!-- Azure Monitor -->
+        <module>sdk/monitor/azure-monitor-opentelemetry-exporter</module>
```

Comment:
What is this for?

Reply:
This is for generating an aggregate javadoc that includes all client libraries in a single javadoc [index.html](https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-java/staging/apidocs/index.html)